src/main/java/com/notably/logic/suggestion/SuggestionEngineImpl.java

```diff
@@ -31,6 +31,7 @@
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
 
     private static final List<String> COMMAND_LIST = List.of("new", "edit", "delete", "open", "help", "exit", "search");
+    private static final List<String> SHORTHANDS_LIST = List.of("n", "e", "d", "o", "h", "s");
```

Comment:
Doesn't seem like this is used anywhere. Any intention to use this list in the future?

Reply:
Woops, I intended to use it but forgot that in the end I changed my implementation. Thanks for pointing out!